siteimprove/admin/js/siteimprove.js

```diff
@@ -121,6 +121,11 @@
       if( this.method === "domain" ){
         _si.push(['input', this.url, this.token, function() { console.log('Inputted new javascript overlay file'); } ]); 
```

Comment:
This `if` was not there in v1 and the whole console log that happens here makes we wonder if this is some left-over debugging code that we don't need anymore.